test/e2e/common/downward_api.go

```diff
@@ -183,6 +183,82 @@ var _ = framework.KubeDescribe("Downward API", func() {
 
 		testDownwardAPIUsingPod(f, pod, env, expectations)
 	})
+
+})
+
+var _ = framework.KubeDescribe("Downward API [Serial] [Disruptive] [Feature:LocalStorageCapacityIsolation]", func() {
+	f := framework.NewDefaultFramework("downward-api")
+
+	Context("Downward API tests for local ephemeral storage", func() {
+		BeforeEach(func() {
+			framework.SkipUnlessLocalEphemeralStorageEnabled()
+		})
+
+		It("should provide container's limits.ephemeral-storage and requests.ephemeral-storage as env vars [Feature:LocalStorageCapacityIsolation]", func() {
```

Comment:
It seems that `[Feature:.*]` here is redundant with `Context` description above.

Reply:
yeah, i added `[Feature:.*]` first but it did not work, so i added `Context` later

Reply:
The `Context` text is not written to the output at the same time as `Describe` and `It` text, but you will see it in the test result, and the context text is included in the `focus=` filter. I move common `It` text to the parent `Context` description just so it isn't repeated. But what you have here works fine too.

Reply:
My above comment has not been addressed.